pkg/tasks/machine_deployments.go

```diff
@@ -34,7 +34,13 @@ func createMachineDeployments(s *state.State) error {
 		return nil
 	}
 
+	if !s.CreateMachineDeployments {
+		s.Logger.Info("Skipped creating MachineDeployments...")
+		return nil
+	}
+
 	s.Logger.Infoln("Creating worker machines...")
+	s.Logger.Warnln("KubeOne will not manage MachineDeployments objects besides creating one time...")
```

Comment:
The problem is that this is not completely true. KubeOne will also upgrade MDs for you if you choose so. We also talked about providing a link to the docs that describe how MDs work.

1) I would phase the message such as "KubeOne will not manage MachineDeployments objects besides initially creating them and optionally upgrading them...".
2) Linking to a guide such as Using machine-controller (https://docs.kubermatic.com/kubeone/v1.3/guides/machine_controller/) could be very beneficial to users. Although, that guide should be refreshed a bit.

Reply:
:heavy_check_mark: